02B-Python-Omop-walkthrough-critical-care/02B-Python-Omop-walkthrough-critical-care.md

```diff
@@ -0,0 +1,2006 @@
+This document is an introductory walkthrough, demonstrating how to read into Python some OMOP data from a series of csv files, join on concept names and explore and visualise the data.
+
+It uses synthetic data created in UCLH as a part of the  [Critical Care Health Informatics Collaborative (CCHIC) project](https://safehr-data.github.io/uclh-research-discovery/projects/uclh_cchic_s0/index.html).
+
+# Installing & loading required packages
+1. ### pip install pandas
+
+2. ### pip install seaborn
+
+3. ### pip install matplotlib
```

Comment:
Happy with this documentation for now, we can tidy it up later so it we can run the notebook in CI